projects/showcase/src/assets/css/google_icon_material.css

```diff
@@ -0,0 +1,23 @@
+/* fallback */
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(https://fonts.gstatic.com/s/materialicons/v139/flUhRq6tzZclQEJ-Vdg-IuiaDsNc.woff2) format('woff2');
+}
+
+.material-icons {
+  font-family: 'Material Icons';
+  font-weight: normal;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
+  -webkit-font-feature-settings: 'liga';
+  -webkit-font-smoothing: antialiased;
+}
```

Comment:
Yes @semicolin  It is necessary because if I remove this it doesn't work properly.
for reference, I attached a screenshot.
![image](https://user-images.githubusercontent.com/117890382/214529849-9cacf3eb-e115-4de2-9215-3418220d778e.png)